frb_example/pure_dart_multi/dart/valgrind_util.py

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env python3
```

Comment:
Hey, do not copy and paste blindly :) Follow the DRY rule

Moreover, do we really need to run valgrind for pure-dart-multi? or is a simple `dart test` sufficient?

If the latter, remove run.sh and valgrind_util.py